src/functions/assertions/BeNullOrEmpty.ps1

```diff
@@ -52,7 +53,8 @@ function Should-BeNullOrEmpty([object[]] $ActualValue, [switch] $Negate, [string
             $failureMessage = NotShouldBeNullOrEmptyFailureMessage -Because $Because
         }
         else {
-            $failureMessage = ShouldBeNullOrEmptyFailureMessage -ActualValue $ActualValue -Because $Because
+            $valueToFormat = if ($singleValue) { $expandedValue } else { $ActualValue }
```

Comment:
This seems to affect non empty hashtables too. Shall we add a unit tests to verify the exception message when the hast table is not empty? So it too returns the expected message? There is an existing test at line 22-24 but it does not verify the exception message.

Reply:
We don't do any special formatting for hashtables. It only drops the outer array, which in a single-item case should be the array Pester wrapped pipeline-input in.

```ps1
@{ hello = "world" } | Should -BeNullOrEmpty
# Pester 5.3.3
Expected $null or empty, but got @(System.Collections.Hashtable).

# After PR
Expected $null or empty, but got System.Collections.Hashtable

Reply:
Looks good.